Research/Data Analysis Notebook.ipynb

```diff
@@ -0,0 +1,3555 @@
+       "3709     run    pass   kneel down  \n",
+       "3710     run    pass   kneel down  \n",
+       "3711     run    pass   kneel down  \n",
+       "3712     run    pass   kneel down  \n",
+       "3713     run    pass   kneel down  \n",
+       "3714     run    pass   kneel down  \n",
+       "3715     run    pass   kneel down  \n",
+       "3716     run    pass   kneel down  \n",
+       "3717     run    pass   kneel down  \n",
+       "3718     run    pass   kneel down  \n",
+       "3719     run    pass   kneel down  \n",
+       "3720     run    pass   kneel down  \n",
+       "3721     run    pass   kneel down  \n",
+       "3722     run    pass   kneel down  \n",
+       "3723     run    pass   kneel down  \n",
+       "3724     run    pass   kneel down  \n",
+       "3725     run    pass   kneel down  \n",
+       "3726     run    pass   kneel down  \n",
+       "3727     run    pass   kneel down  \n",
+       "3728     run    pass   kneel down  \n",
+       "3729     run    pass   kneel down  \n",
+       "\n",
+       "[3730 rows x 14 columns]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cdata"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{' punt'}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "set(cdata.option1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{' kick a field goal'}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "set(cdata.option2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{' run'}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "set(cdata.option3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{' pass'}"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "set(cdata.option4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{' kneel down'}"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type
```

Comment:
Definitely will read the material you linked and use '.str.' more frequently. 

I believe I got rid of the prior step to this to determine what record had the longest text length for the orig_antecedent column. (There are better ways of going about this but I was just pushing through some ideas)

This column contains the football 'scenario' that the survey respondents needed to analyze and pick a response. But not all of the scenarios are very detailed. I pulled the longest one out and split the sentences apart into a list: 
` ['It is second down and inches',
 " The ball is on your opponent's 5 yardline",'
 ' There is 3 seconds left in the second quarter',
 ' You are down by 3 points',
 ' Would you:']`

The last line is erroneous but the remaining items contain scenario information that I want to parse out.